<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prueba Juan_Obando</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/Recursos/iconfinder_calculator-math-tool-school_2824440.png</normaloff>:/Recursos/iconfinder_calculator-math-tool-school_2824440.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>GroupBox</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Modelo</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3" colspan="3">
           <widget class="QRadioButton" name="inBlancoNegro">
            <property name="text">
             <string>Blanco y Negro</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="5">
           <widget class="QLineEdit" name="inModelo"/>
          </item>
          <item row="5" column="0" colspan="3">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Precio de compra</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3" colspan="3">
           <widget class="QLineEdit" name="inPrecio"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Marca</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="4">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Contador de impresiones</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2" colspan="4">
           <widget class="QLineEdit" name="inNumSerie">
            <property name="maxLength">
             <number>7</number>
            </property>
           </widget>
          </item>
          <item row="4" column="4" colspan="2">
           <widget class="QSpinBox" name="inContador">
            <property name="maximum">
             <number>999999</number>
            </property>
            <property name="stepType">
             <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="5">
           <widget class="QLineEdit" name="inMarca"/>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>NÃºmero de serie</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QRadioButton" name="inColor">
            <property name="text">
             <string>Color </string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>GroupBox</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTableWidget" name="outResumen"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="OkpushButton">
         <property name="text">
          <string>Insertar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>Showcard Gothic</family>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>TOTAL:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="outTotal">
            <property name="font">
             <font>
              <family>Palatino Linotype</family>
              <pointsize>20</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(236, 0, 0);
</string>
            </property>
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>https://github.com/JFernandoObando/Prueba</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
      <property name="buddy">
       <cstring>https://github.com/JFernandoObando/Prueba</cstring>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionGuardar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Recursos/disco-flexible.png</normaloff>:/Recursos/disco-flexible.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
